test/e2e/nodenetworkstate_conditions_test.go

```diff
@@ -0,0 +1,92 @@
+package e2e
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+
+	corev1 "k8s.io/api/core/v1"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+)
+
+var _ = Describe("NodeNetworkStateCondition", func() {
+	var (
+		br1Up = nmstatev1alpha1.State(`interfaces:
+  - name: eth1
```

Comment:
It is not necessary to configure eth1, configuring bridge alone should be enough (yes, we do it all over this project and it is wrong)